From bc62215d60a1b956474cd821e9bd08b4b48dafab Mon Sep 17 00:00:00 2001
From: Jernej Skrabec <jernej.skrabec@gmail.com>
Date: Sun, 8 Oct 2023 16:18:18 +0200
Subject: [PATCH] VideoLayerBridgeDRMPRIME: Remove pitch check

With some formats, like single plane and/or compressed ones, pitch can
be 0, since it has no meaning there.

In any case, drmModeAddFB2WithModifiers() will verify arguments,
including pitch (if applicable), and reject invalid combinations.
---
 .../VideoRenderers/HwDecRender/VideoLayerBridgeDRMPRIME.cpp     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VideoLayerBridgeDRMPRIME.cpp b/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VideoLayerBridgeDRMPRIME.cpp
index 233e6310bb823..34d1ab6235591 100644
--- a/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VideoLayerBridgeDRMPRIME.cpp
+++ b/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VideoLayerBridgeDRMPRIME.cpp
@@ -105,7 +105,7 @@ bool CVideoLayerBridgeDRMPRIME::Map(CVideoBufferDRMPRIME* buffer)
   {
     int object = layer->planes[plane].object_index;
     uint32_t handle = buffer->m_handles[object];
-    if (handle && layer->planes[plane].pitch)
+    if (handle)
     {
       handles[plane] = handle;
       pitches[plane] = layer->planes[plane].pitch;
